src/components/ItaliaTheme/Blocks/Listing/CardWithSlideUpTextTemplate.jsx

```diff
@@ -76,8 +76,14 @@ const CardWithSlideUpTextTemplate = (props) => {
                     <p>{item.description}</p>
                   )}
                   <div className="read-more">
+                    {/*
+                    because of https://developer.chrome.com/blog/migrate-way-from-data-urls-in-svg-use
+                    current implementation of Icon in CardReadMore component is not working.
+                    we disable it for now.
+                    iconName="it-arrow-right"
+                    */}
                     <CardReadMore
-                      iconName="it-arrow-right"
+                      iconName={false}
```

Comment:
```suggestion
              
```

il default per iconName è 'it-arrow-right' 
lo toglierei semplicemente.
se mettiamo {false} non viene disegnata l'icona (che ora si vedeva)

Reply:
la richiesta di modifica è proprio perchè l'icona non si vede e da molti errori in console, almeno qui:
https://www.auslromagna.it/
![image](https://github.com/user-attachments/assets/afa75eef-6d5f-46de-a038-75856636ea1c)

Reply:
ah ok. occhio che il CardReadMore è usato in tanti posti.. 
lo fisserei ovunque è usato a questo punto..

Reply:
@giuliaghisini io intanto chiuderei questa.